docs/spark/tutorials/databricks-deployment.md

```diff
@@ -14,10 +14,11 @@ This tutorial teaches you how to deploy your app to the cloud through Azure Data
 In this tutorial, you learn how to:
 
 > [!div class="checklist"]
-> Create an Azure Databricks workspace.
-> Publish your .NET for Apache Spark app.
-> Create a Spark job and Spark cluster.
-> Run your app on the Spark cluster.
+>
+> - Create an Azure Databricks workspace.
```

Comment:
I think these need to be in a list for the checklist markup to work

Reply:
Thanks!